code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -1,5 +1,5 @@
 /obj/item/weapon/gun/projectile/automatic/pistol
-	name = "\improper FK-69 pistol"
+	name = "stechkin pistol"

```

Comment:
And... Walther PPK have a 10mm modification!
Why we don't rename it to Walther?